spec/spec_helper.rb

```diff
@@ -2,9 +2,6 @@
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
 SimpleCov.minimum_coverage 100
-SimpleCov.start do
-  add_filter '/.bundle/'
-end
```

Comment:
Right after we discussed it, I noticed that this one had gone through with an earlier PR.  I'm happy to continue to bundle to `vendor/bundle` which seems to fix it on my system without requiring any configuration.